api-test/internal/conformance/client.go

```
@@ -0,0 +1,334 @@
+	// Redact at the capture point, not downstream: these calls are dropped from
+	// the report today, but the suite admin token must never depend on that.
+	if _, ok := call.ReqHeaders["Authorization"]; ok {
+		call.ReqHeaders["Authorization"] = []string{"***redacted***"}
+	}
+	resp, err := c.http.Do(req)
+	if err != nil {
+		call.RespBody = "ERROR: " + err.Error()
+		c.calls = append(c.calls, call)
+		return nil, 0, err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)